compute/cloud-client/instances/test/instancesTest.php

```diff
@@ -170,4 +169,25 @@ public function testSetUsageExportBucketCustomPrefix()
         ]);
         $this->assertStringContainsString('project `' . self::$projectId . '` is disabled', $output);
     }
+    public function testListAllImages()
+    {
+        $output = $this->runFunctionSnippet('list_all_images', [
+            'projectId' => 'windows-sql-cloud',
+        ]);
+
+        $this->assertStringContainsString('sql-2012-enterprise-windows', $output);
+        $arr = explode(PHP_EOL, $output);
+        $this->assertGreaterThanOrEqual(2, count($arr));
+    }
+    public function testListImagesByPage()
```

Comment:
nit: we want new lines between functions
```suggestion

        public function testListImagesByPage()
```